install_resources.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
why not install them as module with maven, by putting them at the top of the <modules> list in the parent pom.xml?